media_kit/lib/src/player/web/player/real.dart

```diff
@@ -368,6 +380,27 @@ class WebPlayer extends PlatformPlayer {
     }
   }
 
+  void loadResource(src) {
```

Comment:
Is checking for .m3u8 in the URL sufficient? Are there any other checks we must have?

Maybe package:video_player_web_hls can serve as an example:

https://github.com/balvinderz/video_player_web_hls/blob/main/lib/src/video_player.dart#L319-L359

Reply:
> Is checking for .m3u8 in the URL sufficient? Are there any other checks we must have?
> 
> Maybe package:video_player_web_hls can serve as an example:
> 
> https://github.com/balvinderz/video_player_web_hls/blob/main/lib/src/video_player.dart#L319-L359

current i have no better idea. and the url can't be opened;

Reply:
I updated the URL in-case you decide to have a look.

Reply:
> I updated the URL in-case you decide to have a look.

found it

Reply:
code added, but developer can't pass headers currently;